carRental/src/main/java/com/amigoscode/chohort2/carRental/driverLicense/DriverLicenseController.java

```diff
@@ -0,0 +1,11 @@
+package com.amigoscode.chohort2.carRental.driverLicense;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/v1/driver_licenses")
```

Comment:
It's highly recommended to use hyphens ( **-** ) instead of underscores ( **_** )in any url-like lines;

Reply:
done